app/templates/index.html

```diff
@@ -240,15 +240,15 @@ <h2>Nigeria Se4All</h2>
         </div>
```

Comment:
The header is missing

Reply:
It is missing, I didn't understant that it should be part of the footer too.

Reply:
Not part of the footer, but part of a header

Reply:
It is defined already but hidden

Reply:
Alright

Reply:
I added the logos to the header